kpi/models/asset.py

```diff
@@ -239,6 +243,12 @@ def _unlink_list_items(self, content):
             if '$prev' in row:
                 del row['$prev']
 
+    def _revert_kobo_lock_structre(self, content):
+        revert_kobo_lock_structre(content)
+
+    def _strip_kobo_locking_profile(self, content):
+        strip_kobo_locking_profile(content)
```

Comment:
i wouldn't follow the example of `_remove_empty_expressions()` here: these methods, which are really static methods, don't seem to add anything. unless i'm missing something, directly call the functions that were imported from formpack.

Reply:
I just followed the structure that existed in the class, but I'll change it to something more sensible 😄